### ISSUE TITLE
Anyone know of a maintained fork of this?

### ISSUE THREAD

**Issue by GregJohnStewart** [kohsuke/args4j#170]
Love the library and to give it credit, still works well after all this time, but it is starting to fall behind in terms of newer java versions (not to mention all the unaddressed issues here on GitHub).

Anyone know of a fork of this that is maintained and accessible on a maven repo?

I see a bunch of forks, but need the 'on Maven and maintained' part of it. Also willing to fork it myself and get it maintained but would need to get some help (Setting it up with Maven, help finding/ fixing issues, etc. Would just be nice to see a community behind the work).

Apologies to kohsuke that this is put in the issues of the OG repo, but unsure where else to put it. I would love to help maintaining this (original) repo if it seemed like it was still being updated/ accepting pull requests.

**Comment by apj68** [comment]
I'd be willing to help, I've used this quite a bit over the years and don't know of another CLI library that I prefer.  I just came here to log a bug but it does look like it's falling into the abandoned territory.

**Comment by erik-wramner** [comment]
I can also put in some work now and then. Ideally without forking if someone could get commit rights to this repo to approve pull requests.

**Comment by thomas-mc-work** [comment]
I've migrated to https://picocli.info/. It has a very similar approach compared to args4j:

    @Command(name = "checksum", mixinStandardHelpOptions = true, version = "checksum 4.0",
             description = "Prints the checksum (MD5 by default) of a file to STDOUT.")
    class CheckSum implements Callable<Integer> {

        @Parameters(index = "0", description = "The file whose checksum to calculate.")
        private File file;

        @Option(names = {"-a", "--algorithm"}, description = "MD5, SHA-1, SHA-256, ...")
        private String algorithm = "MD5";

**Comment by apj68** [comment]
Yeah, I recently came across picocli, too.  I've been using it rather than args4j, it seems to have everything I need and more.

**Comment by markkolich** [comment]
@kohsuke are you still actively maintaining this library?

**Comment by erik-wramner** [comment]
@markkolich considering how many times that question has been asked here without an answer the last few years I would say the answer is no.

**Comment by markkolich** [comment]
@erik-wramner he has been active in other projects, namely https://github.com/github-api/github-api which is why I was thinking an explicit @-mention was warranted.
